CONTRIBUTING.md

```diff
@@ -51,6 +52,10 @@ Pull requests should only include `vendor/` updates if they are part of the same
 
 The `vendor/` update needs to be justified as part of the PR description, and must be verified by the reviewers and/or merger to always reference an existing upstream commit.
 
+## Translation
+
+We do all translation work inside [CrowdIn](https://crowdin.com/project/gitea). The only translation that is maintained in this git repository is [`en_US.ini`](https://github.com/go-gitea/gitea/blob/master/options/locale/locale_en-US.ini) and is synced regularily to CrowdIn. Once a translation has reached A SATISFACTORY PERCENTAGE it will be synced back into this repo and included in the next released version.
```

Comment:
Minor, but the website's name seems to be stylized as `Crowdin`, not `CrowdIn` (https://www.google.com/search?q=crowdin)

Reply:
Done 🙂